src/utilities/PrivateRoute.js

```diff
@@ -7,13 +7,21 @@ import { Redirect, Route } from 'react-router-dom';
 export const PrivateRoute= ({ component: Component, ...rest }) => {
     const { keycloak, initialized } = useKeycloak();
 
+    const isMember = ( keycloak ) => {
+      const userRoles = keycloak.realmAccess.roles  
+      const roles = ["github-org-bcgov", "github-org-bcgov-c", "github-org-bcdevops", "idir-user"];
```

Comment:
hard coded configuration value. Should be passed in as config. This can be represented by environment variables or static config. 
eg:

`REACT_APP_AUTHORIZED_SSO_ROLES="github-org-bcgov,github-org-bcgov-c,foo,mattdamon"`


```constants.js
const AUTHORIZED_ROLES = process.env.REACT_APP_AUTHORIZED_SSO_ROLES.split(',');

```